frontstage/views/secure_messaging.py

```diff
@@ -389,4 +381,12 @@ def message_get(session, msg_id):
         raise ExternalServiceError(get_message)
     message = json.loads(get_message.text)
     loggerb.debug('Retrieved message successfully')
+
+    loggerb.debug('Attempting to remove unread label')
+    data = {"label": 'UNREAD', "action": 'remove'}
+    url = app.config['MESSAGE_MODIFY_URL'].format(msg_id)
+    response = requests.put(url, data=json.dumps(data), headers=headers)
+    if response.status_code != 200:
+        loggerb.error("Failed to remove unread label")
+
```

Comment:
Do you not want to raise ExternalSerivceError if you can't modify the message status?

Reply:
I've decided not to as I don't think we want to kick the user out over such a small thing